frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "node": ">=14"
   },
   "scripts": {
-    "build": "vite build",
+    "build": "tsc && vite build",
```

Comment:
The downside of this and why we might want to discard this is because it introduces an overhead and `tsc` is already executed in the `test` script.

Reply:
If we are adding it to the build, I think it's a reasonable thing to do.

Reply:
:heavy_check_mark:

Reply:
Should be fine, however do (or should) we really build the frontend that often when developing locally? I believe `tsc` is already checked in `ts:check` on the workflows.

Reply:
Not in main. It's on the PRs but if we merge a PR that breaks main, main will still report as green, so that might be actually more important

Reply:
Should `yarn build` also build the frontend? Or should this be an additional workflow step of GH actions?

Reply:
https://github.com/Unleash/unleash/pull/3049/commits/99c567dc5f7b1d60a3ca5c202b20e4b1b7b9c452 maybe?

Reply:
That didn't work (you can see the failed builds)